geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloFeatureSource.scala

```diff
@@ -228,7 +232,7 @@ class DelegatingResourceInfo(source: SimpleFeatureSource) extends ResourceInfo {
 
   import scala.collection.JavaConversions._
 
-  private val keywords = Collections.unmodifiableSet(Set("features", getName))
+  private val keywords: java.util.Set[String] = new java.util.HashSet[String](List("features", getName))
```

Comment:
@elahrvivaz Running into issues with having it as unmodifiable. I get an UnsupportedOperationException when I try to combine source.getSchema.getKeywords with Set("features", getName). Even making it explicit like

    val l = source.getSchema.getKeywords  

    l.addAll(List("features", getName))      

    Collections.unmodifiableSet(l)

still throws the error.


Would it be reasonable to have "features" and getName be returned from source.getSchema.getKeywords?